cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1059,6 +1059,8 @@ def _add_head_node(self):
             indent=4,
         )
 
+        cloudformation_url = f"https://cloudformation.{self.stack.region}.{get_url_domain_suffix()}"
```

Comment:
how this was working in CN?
Before, we weren't setting the CloudFormationUrl for CN, now we are setting the URL based on the get_url_domain_suffix. Is this a change?

Reply:
I think the change is safe.
CloudFormation endpoints in the supported China regions are:
1. cn-north-1: cloudformation.cn-north-1.amazonaws.com.cn
2. cn-northwest-1: cloudformation.cn-northwest-1.amazonaws.com.cn

Reply:
Will execute a manual test (create+update) in China to double check my assumptions in any case and let you know.